server/service/src/sync/test/integration/transfer/requisition.rs

```diff
@@ -105,6 +106,7 @@ async fn integration_sync_requisition_transfers_initialisation() {
             &response_site.store,
             &item1,
             &item2,
+            6,
```

Comment:
Any particular reason for `6` here? Would 0/1 be better or make it optional? Sorry I added this in but didn't update the integration test...

Reply:
No particular reason XD. Optional would probably be better